File: python/swagger_client/models/transaction_type.py
# coding: utf-8

class TransactionType(object):
    SALE = "Sale"
    CREDIT = "Credit"
    AUTHORIZATION = "Authorization"
    PREAUTH = "PreAuth"
    POSTAUTH = "PostAuth"
    FORCEAUTH = "ForceAuth"
    VOID = "Void"
    QUERY = "Query"
    VERIFY = "Verify"
    MARK = "Mark"
    RETURN = "Return"
    USE = "Use"

    swagger_types = {
    }

    attribute_map = {
    }

    def __init__(self):  # noqa: E501
        """TransactionType - a model defined in Swagger"""  # noqa: E501
        self.discriminator = None

File: php/SwaggerClient-php/lib/Model/TransactionType.php
<?php

namespace Swagger\Client\Model;

use \Swagger\Client\ObjectSerializer;

class TransactionType
{
    const SALE = 'Sale';
    const CREDIT = 'Credit';
    const AUTHORIZATION = 'Authorization';
    const PRE_AUTH = 'PreAuth';
    const POST_AUTH = 'PostAuth';
    const FORCE_AUTH = 'ForceAuth';
    const VOID = 'Void';
    const QUERY = 'Query';
    const VERIFY = 'Verify';
    const MARK = 'Mark';
    const _RETURN = 'Return';
    const _USE = 'Use';

    public static function getAllowableEnumValues()
    {
        return [
            self::SALE,
            self::CREDIT,
            self::AUTHORIZATION,
            self::PRE_AUTH,
            self::POST_AUTH,
            self::FORCE_AUTH,
            self::VOID,
            self::QUERY,
            self::VERIFY,
            self::MARK,
            self::_RETURN,
            self::_USE,
        ];
    }
}
